src/pages/preferences/preferencesTabs/NotificationsTab.tsx

```diff
@@ -89,6 +93,12 @@ export const NotificationsTab: React.VFC = () => {
           value={slackUsernameField}
           data-cy="slack-username-field"
         />
+        <StyledTextInput
+          label="Slack Member Id"
```

Comment:
nit. I prefer this with capital `ID`

```suggestion
          label="Slack Member ID"
```